src/resources/nftOffers/index.ts

```diff
@@ -0,0 +1,31 @@
+import { arcClient } from '@/graphql';
+import { GetNftOffersQuery, SortCriterion } from '@/graphql/__generated__/arc';
+import { createQueryKey } from '@/react-query';
+import { useQuery } from '@tanstack/react-query';
+
+const STALE_TIME = 60000; // 1 minute
```

Comment:
using a pretty low stale time bc offers are pretty spammy in general. nfts i've looked at have TONS of offers, and many of them expire quickly

Reply:
may be worth noting here or in linear that we should probably change this for prod, maybe im tripping but hitting this every min is expensive

Reply:
Yeah agree, if a user wants up-to-the-second resolution they should probably use a purpose-built solution. Maybe a discussion, but like every 3-5min seems better?